data/membership-schema/src/main/resources/net/corda/schema/membership/member/static/registration/1.0/corda.member.static.registration.json

```diff
@@ -17,6 +17,11 @@
       "description": "Version supported for the flow protocol used by the notary service. Valid only when one of the roles is notary.",
       "type": "integer",
       "minimum": 1
+    },
+    "^ext\\.[a-zA-Z0-9.]+$": {
+      "description": "Optional. A user can specify additional properties, they must be prefixed with `ext.`.",
```

Comment:
Maybe either
`Optional. A user can specify additional properties, which must be prefixed with `ext.`.`
or
`Optional. A user can specify additional properties. They must be prefixed with `ext.`.`